package.json

```diff
@@ -88,8 +89,7 @@
       "**/@types/*/**",
       "**/grunt-*",
       "**/grunt-*/**",
-      "x-pack/typescript",
-      "kbn_tp_*/**"
+      "x-pack/typescript"
```

Comment:
@ThiagoTcorsi Wanted to get your opinion on this change. Typescript was giving @spalger and I some trouble and this seemed necessary to resolve it. Is there anything we might have inadvertently broke?

Reply:
@mattkime as far as I remembered, in the past we had the need to add `"kbn_tp_*/**"` to the yarn workspaces nohoist feature in order to have the node_modules installed for the `kibana/test/plugin_functional/plugins`, otherwise those plugins during the tests can't read the node_modules they declare from the dll bundle

Reply:
@mistic so if it passes, its good?

Reply:
@mattkime well maybe I'm missing something here, but what have changed in order to avoid the dependencies for that plugins being hoisting by the yarn workspaces?